client/extensions/woocommerce/app/products/product-form-variations-table.js

```diff
@@ -93,22 +93,23 @@ class ProductFormVariationsTable extends React.Component {
 	render() {
 		const { variations, translate } = this.props;
 		return (
-			<div className="products__product-form-variation-table-wrapper">
-				<table className="products__product-form-variation-table">
-					<thead>
-						<tr>
-							<th></th>
-							<th></th>
-							<th className="products__product-price">{ translate( 'Price' ) }</th>
-							<th>{ translate( 'Dimensions & weight' ) }</th>
-							<th>{ translate( 'Manage stock' ) }</th>
-						</tr>
-					</thead>
-					<tbody>
-						{ variations && variations.map( ( v ) => this.renderVariationRow( v ) ) }
-					</tbody>
-				</table>
-				{ this.renderModal() }
+			<div className="products__product-form-variation-table-shadow">
+				<div className="products__product-form-variation-table-wrapper">
+					<table className="products__product-form-variation-table">
+						<thead>
+							<tr>
+								<th></th>
+								<th className="products__product-price">{ translate( 'Price' ) }</th>
+								<th>{ translate( 'Dimensions & weight' ) }</th>
```

Comment:
Hi! I've found a possible matching string that has already been translated **77** times:
`translate( 'Dimensions' )` *ES Score: 10*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/try/woocommerce-remove-visible-checkbox)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).